source/docs/articles/drupal/8.md

```diff
@@ -0,0 +1,55 @@
+---
+use: [8]
+layout: default
+title: Drupal 8 on Pantheon
+---
+
+<div class="container article">
+  <div class="col-md-9">
+  <h1>Drupal 8 Docs</h1>
+  {% for article in data.8 %}
+  <article>
+  <h3><a href="{{ article.url }}">{{ article.title }}</a></h3>
+  <p> {{ article.description }} </p>
+  </article>
+  {% endfor %}
+  <hr>
+<h2> Install Drupal 8 </h2>
```

Comment:
Yep, just making sure that everything in the doc gets preserved elsewhere, so I haven't deleted it yet. But agreed :+1: It will look similar to the guides directory list page